Source/Charts/Charts/PieChartView.swift

```diff
@@ -128,6 +128,26 @@ open class PieChartView: PieRadarChartViewBase
         
         drawMarkers(context: context)
     }
+
+    /// if width is larger than height
+    private var widthLarger: Bool
```

Comment:
I believe there is already an extension on `CGSize` and `CGRect` to provide an aspect ratio which would simplify this to
```
private var widthLarger: Bool { _viewPortHandler.contentRect.aspect > 1
```

Reply:
It could be useful to provide an orientation instead
```
enum Orientation { case portrait, landscape }

extension CGSize {
    var orientation: Orientation { aspect > 1 }
}

extension CGRect {
    var orientation: Orientation { size.orientation }
}
```

Reply:
Hi @jjatie sorry for the deplay. Finally got a little time to work on this. I searched it seems current master didn't have  aspect?

Reply:
@jjatie mind take a look recently? I want to merge this and release a new build for Swift 5.2. I didn't see `aspect` so I use the width > height for now

Reply:
@jjatie plan to merge this this week and release new build for Swift 5.2.

Would you mind take a look?